samples/apps/copilot-chat-app/webapp/src/libs/useChat.ts

```diff
@@ -146,14 +160,22 @@ export const useChat = () => {
                 getEnabledPlugins(),
             );
 
+            // When a document is uploaded we only save the user's message to storage, and
+            // do not generate a bot response. Therefore, we do not need to update the conversation.
+            if (type === ChatMessageType.Document) {
+                return;
+            }
+
+            const isPlan = PlanUtils.isPlan(result.value);
             const messageResult = {
+                type: isPlan ? ChatMessageType.Plan : ChatMessageType.Message,
```

Comment:
MessageType should be returned either in the context or chat message result, right? Can we just assign that value here?

Reply:
good point. this is now returned in the context.